afterupdate.marker

```diff
@@ -0,0 +1 @@
+# If this file is here, the afterupdate scripts did not run.
```

Comment:
It is exactly the opposite. If the file is there we just update and afterupdate scripts will be executed.

Anyways you can keep the file empty, since nobody is expected to read the file.

Reply:
Let us hope, nobody reads the file. 
A short comment inside the file does not hurt. The content is just a reminder, to not forget, what the file is good for.